components/Header.tsx

```diff
@@ -59,10 +59,10 @@ function Header() {
   );
 
   return (
-    <>
+    <div className="fixed z-30">
```

Comment:
why gi addan div?

Reply:
naa koy gi absolute nga pic, and I mu overlap siya ig wala gi set ang z layer sa outer div. Here's the pic kung walay outer z layer div:
<img width="369" alt="image" src="https://user-images.githubusercontent.com/82187749/214058100-332b2988-7e83-419a-993d-110d6c22fe07.png">